packages/rocketchat-file-upload/server/methods/getS3FileUrl.js

```diff
@@ -0,0 +1,35 @@
+const crypto = Npm.require('crypto');
+let protectedFiles, S3accessKey, S3secretKey, S3expiryTimeSpan;
+
+RocketChat.settings.get('FileUpload_ProtectFiles', function(key, value) {
+	protectedFiles = value;
+});
+
+RocketChat.settings.get('FileUpload_S3_AWSAccessKeyId', function(key, value) {
+	S3accessKey = value;
+});
+
+RocketChat.settings.get('FileUpload_S3_AWSSecretAccessKey', function(key, value) {
+	S3secretKey = value;
+});
+
+RocketChat.settings.get('FileUpload_S3_URLExpiryTimeSpan', function(key, value) {
+	S3expiryTimeSpan = value;
+});
+
+Meteor.methods({
```

Comment:
Please check the use of spaces/tabs and fix this file. You can run your code through eslint, as stated here: https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md